m4macros/Makefile.am

```diff
@@ -1,5 +0,0 @@
-uinstalled_m4 = \
```

Comment:
There was a typo here, so it probably never did anything.

Reply:
Indeed. That means these 2 m4 files were not in the distribution tarball and if that never made any problem to a packager, then that indeed proves these were unneeded, I guess.
So I supposed we could just merge this pull request.